src/main/java/com/backbase/ct/bbfuel/setup/CapabilitiesDataSetup.java

```diff
@@ -262,12 +264,18 @@ private void ingestPockets() {
 
                     this.loginRestClient.login(retailUser.getExternalId(), retailUser.getExternalId());
                     userContextPresentationRestClient.selectContextBasedOnMasterServiceAgreement();
-                    this.pocketsConfigurator.ingestPockets(legalEntity.getExternalId());
+                    List<Pocket> createdPockets = this.pocketsConfigurator.ingestPockets(legalEntity.getExternalId());
+                    ingestPocketTransactions(PocketsConfigurator.EXTERNAL_ARRANGEMENT_ORIGINATION_1, createdPockets);
```

Comment:
We need to ingest corresponding transactions to the user's checking/saving accounts, right? Example:
- pocket has a credit transaction of 10 euros from the user's checking account
- the user's checking account should have a debit transaction of 10 euros

Reply:
implemented also those transactions